qemu-nbd is already provided by subpackage qemu-img, so we don't need to
install qemu package.

--- a/alpine-make-vm-image
+++ b/alpine-make-vm-image
@@ -361,7 +361,7 @@
 	if ! grep -q -w "$ROOTFS" /proc/filesystems; then
 		modprobe $ROOTFS
 	fi
-	_apk add -t $VIRTUAL_PKG qemu qemu-img $(fs_progs_pkg "$ROOTFS")
+	_apk add -t $VIRTUAL_PKG qemu-img $(fs_progs_pkg "$ROOTFS")
 fi
 
 #-----------------------------------------------------------------------
